tests/expr/sumh_literal_test.py

```diff
@@ -0,0 +1,18 @@
+from typing import Any
+
+import narwhals as nw
+from tests.utils import compare_dicts
+
+
+def test_sumh_literal(constructor: Any) -> None:
+    data = {"a": [1, 3, 2], "b": [4, 4, 6], "z": [7.0, 8, 9]}
+    df = nw.from_native(constructor(data), eager_only=True)
+    result = df.with_columns(horizonal_sum=nw.sum_horizontal("a", nw.col("b")))
```

Comment:
this and the other one (`test_sumh`) are both testing `Expr.sum_horizontal` - could we put them both in the same file `tests/expr/sum_horizontal_test`?

Reply:
good idea!